app/src/js/components/Rules/overview.js

```diff
@@ -47,7 +49,16 @@ class RulesOverview extends React.Component {
             sortIdx='timestamp'
             bulkActions={this.generateBulkActions()}
             rowId='name'
-          />
+          >
+            <ListFilters>
+              <Search
+                dispatch={this.props.dispatch}
+                action={searchRules}
+                clear={clearRulesSearch}
+                placeholder='Search Rules'
```

Comment:
Can we add a label here for accessibility? I'm adding labels to the filters on the collections and granules pages in my PR.

```suggestion
                placeholder='Search Rules'
                label='Search'
```